build/references/cryptographic-primitives.md

```diff
@@ -62,6 +62,43 @@ Message Signature: `0xb52aa0535c5c48268d843bd65395623d2462016325a86f09420c81f142
 
 Morty was never seen again.
 
+## Signed Messages
+
+A standard for interoperable generic signed messages based on the Bitcoin Script format and Ethereum format.
+
+```txt
+sign(sha256(length(prefix) . prefix . length(message) . message))
```

Comment:
Does `.` here denote the concatenation operator?

Reply:
Yes the `.` is the concatenation operator. I'll change it to `+` or clarity.